### PR TITLE
Added Bigbro !

### PR DESCRIPTION
**Pull request by LionelF**
Added Bigbro point system

### CODE REVIEW

File: responders/bigbro.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        return message[0:2] == 'pt'
+
+    def generate(self, message):
+        words = message.split(" ")
+
+        if words[1] == 'help':
+            return "pt pseudo message"
+
+        return {
+                'content': "\t\t1 point %s pour %s\n" % (words[1], words[2]),
```
Comment (rande) [inline responders/bigbro.py:14]: English my friend!

Comment (LionelF) [inline responders/bigbro.py:14]: I know but it loses here the orginal spirit if translated :/

**Comment by rande** [comment]
you need to rebase this commit cannot merge

**Comment by rande** [comment]
merged manually

**Pull request closed without merging**